scripts/pr-infra-compose.sh

```diff
@@ -38,6 +44,16 @@ if [ "$2" = "up" ] || [ "$2" = "up-and-load-backup" ]; then
             -f "docker/docker-compose.pr-tests.yml" \
             up -d postgres valkey
```

Comment:
As discussed, it's probably simpler to just do (no `while after that`):
```suggestion
            up --wait postgres valkey
```